src/libs/ajax.js

```diff
@@ -76,12 +77,24 @@ const withRetryOnError = _.curry(wrappedFetch => async (...args) => {
   return wrappedFetch(...args)
 })
 
+
+// Captures given course in error message, compares to root addresses in app configuration plus https://storage.googleapis.com/,
+// and returns the root that matches the error.
+const captureRequestFailure = (...args) => {
+  const errorAddress = _.find(v => {
+    return _.includes(v, args[0])
+  },
```

Comment:
nit: this can be a one liner:
```suggestion
  const errorAddress = _.find(v => _.includes(v, args[0]),
```